src/txn/txn.c

```diff
@@ -2948,12 +2948,18 @@ __wt_verbose_dump_txn(WT_SESSION_IMPL *session)
         if ((id = __wt_atomic_loadv64(&s->id)) == WT_TXN_NONE &&
           __wt_atomic_loadv64(&s->pinned_id) == WT_TXN_NONE)
             continue;
+
         sess = &WT_CONN_SESSIONS_GET(conn)[i];
-        WT_RET(__wt_msg(session,
-          "session ID: %" PRIu32 ", txn ID: %" PRIu64 ", pinned ID: %" PRIu64
-          ", metadata pinned ID: %" PRIu64 ", name: %s",
-          i, id, __wt_atomic_loadv64(&s->pinned_id), __wt_atomic_loadv64(&s->metadata_pinned),
-          sess->name == NULL ? "EMPTY" : sess->name));
+        if (sess->isolation == WT_ISO_READ_UNCOMMITTED)
```

Comment:
I think it is better to move this change inside to __wt_verbose_dump_txn_one. We already have checks about isolation levels there.

Reply:
@quchenhao 
done, I'm not sure I understand what you're trying to say, please help review the code again.
thanks.